wgpu/src/lib.rs

```diff
@@ -387,9 +395,9 @@ static_assertions::assert_impl_all!(SurfaceConfiguration: Send, Sync);
 /// This type is unique to the Rust API of `wgpu`. In the WebGPU specification,
 /// [`GPUCanvasContext`](https://gpuweb.github.io/gpuweb/#canvas-context)
 /// serves a similar role.
-#[derive(Debug)]
-pub struct Surface {
+pub struct Surface<'window> {
     context: Arc<C>,
+    _surface: Option<Box<dyn 'window + WgpuSurfaceRequirement>>,
```

Comment:
```suggestion
    _surface: Option<Box<dyn WgpuSurfaceRequirement + 'window>>,
```

Just style preference